port/unix_include/omrportpg.h

```diff
@@ -85,7 +85,7 @@ typedef struct OMRPortPlatformGlobals {
 	char *si_osVersion;
 	uintptr_t vmem_pageSize[OMRPORT_VMEM_PAGESIZE_COUNT]; /** <0 terminated array of supported page sizes */
 	uintptr_t vmem_pageFlags[OMRPORT_VMEM_PAGESIZE_COUNT]; /** <0 terminated array of flags describing type of the supported page sizes */
-	BOOLEAN isRunningInContainer;	
+	uintptr_t sysinfoControlFlags;
```

Comment:
This should use a type whose width does not depend on the target, `uint32_t` or `uint64_t`: probably the former.